common/display/displayplanemanager.cpp

```diff
@@ -800,7 +800,7 @@ void DisplayPlaneManager::EnsureOffScreenTarget(DisplayPlaneState &plane) {
   }
 
   uint64_t modifier = plane.GetDisplayPlane()->GetPreferredFormatModifier();
-  if (plane.IsVideoPlane())
+  if (plane.HasVideoLayer())
```

Comment:
If a plane has video layer, then it should be a video plane when going to VPP
Have you debug whey plane .IsVideoPlane is not true?

Reply:
We called SetVideoPlane(true) when ForceVPP is called

Reply:
Not clear yet, but "SetVideoPlane" only work for extend display.

Reply:
Then we can fix ResetLayers logic,
without checking the layer count to be 1